src/assets/js/play_url.js

```diff
@@ -0,0 +1,50 @@
+(function (window, document) {
+  "use strict";
+
+  var EXERCISES_PATH = "/exercises";
+  var PLAY_HOST = "https://play.rust-lang.org";
+  var RUST_VERSION = "nightly";
+  var HOME_URL = window.location.protocol + "//" + window.location.host + "/";
+
+  document.addEventListener("DOMContentLoaded", function() {
```

Comment:
(Well, I guess we just need to put a call to `playUrl()` inline, after the `body` tag? (And export it out of the function, obviously.))

Still, I don't care much.